config.toml

```diff
@@ -142,7 +142,7 @@ algolia_docsearch = false
 offlineSearch = true
 
 # Enable syntax highlighting and copy buttons on code blocks with Prism
-prism_syntax_highlighting = false
+prism_syntax_highlighting = true
```

Comment:
Unfortunately this change needs to come with a refactoring of the `command` shortcode :(

Reply:
Yeah sure @alexrashed will try to edit command shortcode and will debug this.